lib/maths/CDataFrameCategoryEncoder.cc

```diff
@@ -231,8 +231,12 @@ std::size_t CEncodedDataFrameRowRef::numberColumns() const {
     return m_Encoder->numberEncodedColumns();
 }
 
-CDataFrameCategoryEncoder::CDataFrameCategoryEncoder(CMakeDataFrameCategoryEncoder builder) {
-    m_Encodings = builder.makeEncodings();
+CDataFrameCategoryEncoder::CDataFrameCategoryEncoder(CMakeDataFrameCategoryEncoder& parameters) {
```

Comment:
nit: for consistency can we call this `builder`.

Reply:
done